apps/editors/tests/test_views.py

```diff
@@ -2330,6 +2330,20 @@ def test_viewing_queue(self):
         data = json.loads(r.content)
         eq_(data[str(self.addon.id)], self.editor.display_name)
 
+    def test_platform_text(self):
+        version = Version.objects.create(addon=self.addon, version='0.2')
+        version.created = datetime.today() + timedelta(days=1)
```

Comment:
I checked, and for some reason, it doesn't work without those two lines. Probably because of caching. You can (if you want) replace those two lines by a call to `version.reload()`.